src/cwe_checker_lib/src/intermediate_representation/jmp.rs

```diff
@@ -58,3 +60,35 @@ pub enum Jmp {
         return_: Option<Tid>,
     },
 }
+
+impl fmt::Display for Jmp {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Jmp::Branch(tid) => write!(f, "-> {}", tid),
+            Jmp::BranchInd(expr) => write!(f, "-> {}", expr),
+            Jmp::CBranch { target, condition } => write!(f, "if {} -> {}", condition, target),
```

Comment:
I would prefer writing `jump to` instead of `->`, since it is easier to understand for someone seeing it the first time. At least for me it was a little confusing at first until I understood what the `->` meant.